OpenRA.Game/Player.cs

```diff
@@ -52,8 +52,19 @@ static CountryInfo ChooseCountry(World world, string name)
 				.WithInterface<CountryInfo>().Where(c => c.Selectable)
 				.ToList();
 
-			return selectableCountries.FirstOrDefault(c => c.Race == name)
+			var selected = selectableCountries.FirstOrDefault(c => c.Race == name)
```

Comment:
> 14:17 | (+pchote) Is this the comment about modders intentionally specifying bogus races?
14:17 | (+Phrohdoh) pchote: it’s not bogus, its multiple levels of random
14:17 | (+pchote) abcdefg30: this isnt a problem that needs solving